### PR TITLE
Esperanto Overhaul 2: The Overhaulening

### PR DESCRIPTION
**Pull request by yellowcrash10**
This PR incorporates the changes requested by @Inferus25 during their review of the overhaul.

### CODE REVIEW

File: eo.json [hunk 1]
```diff
@@ -1212,7 +1212,7 @@
 
         "Tutorial.Welcome.StartTutorialHeader" : "Komenci Lernilon",
         "Tutorial.Welcome.StartTutorialDescription" : "Ni translokos vin al baza lernila mondo por helpi vin komenci kun Neos. Vi povas elekti aŭ havi la lernil-mondon publika, permesante al Neos-komunumo aliĝi kaj helpi vin, aŭ lerni memstare.",
-        "Tutorial.Welcome.StartTutorialPublic" : "Mi volas, ke homoj aliĝi kaj helpi min",
+        "Tutorial.Welcome.StartTutorialPublic" : "Mi volas, ke homoj aliĝu kaj helpi min",
```
Comment (Inferus25) [inline eo.json:1215]: Mankis -u en la dua verbo: "...aliĝu kaj _helpu_ min"

**Review by Inferus25 — CHANGES_REQUESTED** [review]
Just a single change that was missing

File: eo.json (hunk 1 above)
Comment (yellowcrash10) [reply eo.json]: Ah! 😅 Thanks for catching that!

**Comment by Inferus25** [comment]
> Thank you for the review! I will begin making corrections later today.
> 
> In addition to these changes, I was also thinking about implementing suggestions from the [Esperanto documentation for Mozilla localizers](https://mozilla-l10n.github.io/styleguides/eo/); specifically the section regarding subjectless verbs like "Loading...". As far as I know, this is the only document describing how to translate software into Esperanto. What are your thoughts?

That's a good proposal!

However, apart from being a usable option, I don't really agree with the "loading..." -> "ŝarĝado..." translation as being more appropriate than the active participle "ŝarĝanta...". Their argument comes somewhat unrelated:
The [linked article](https://bertilow.com/pmeg/gramatiko/verboj_frazroloj/sensubjektaj.html) from PMEG talks about verbs that naturally doesn't have a subject, such as "pluvi, temi, tagiĝi". That kind of verbs indeed doesn't have a subject, neither explicit nor implicit. But that doesn't imply that all the other verbs need to have an _explicit_ subject. When these more common verbs doesn´t seem to have an explicit subject, the subject is what is called a tacit subject, and it is actually very often used in Esperanto. Without it, [these examples](https://bertilow.com/pmeg/gramatiko/i-verboj/kvazau_chefverboj.html) given in PMEG wouldn't exist, for example.

With an active participle "ŝarĝanta..." we could imply an abbreviation of a longer phrase, such as: "La sistemo estas _ŝarĝanta_ la mondon", or even simpler, as an answer to a query: "Kiel statas la sistemo? -_Ŝarĝanta_ la mondon".

"ŝarĝado..." would work for me too. But I would actually suggest we use "ŝarĝanta...". Although it is only a matter of style. 

In the other hand, I would not recommend we use "ŝarĝante..." with -e, because of the nature of the -e participles being closer to an actual gerund. I would find it harder to advocate for an implied sentence or another justification in that situation.

**Comment by Frooxius** [comment]
I was about to merge it, but looks like you need to resolve a few things first? I'll keep it open, let me know once you're happy with this to be merged.

**Comment by Inferus25** [comment]
Yes, thank you! We will let you know when it is ready to merge

**Comment by Inferus25** [comment]
@yellowcrash10, saluton.
What are your thoughts?

**Comment by yellowcrash10** [comment]
@Inferus25 I agree that the -e ending doesn't fit very well. I'll change those to -anta.

It appears there are also some new strings to translate, so I'll just add those to this PR.

File: eo.json [hunk 2]
```diff
@@ -1382,6 +1382,20 @@
         "Tutorial.Intro.HelpSite" : "Helpo",
 
         "Object.Mirror.Toggle" : "Loka Baskulo",
+        
+        "Tutorial.Streaming.StageControl" : "Sceneja Regilo",
+        "Tutorial.Streaming.StageUI" : "Baskuligi Scenejan UI",
+        "Tutorial.Streaming.StageLights" : "Baskuligi Scenej-lumojn",
+        "Tutorial.Streaming.StageLightController" : "Baskuligi Lum-regilon",
+        "Tutorial.Streaming.StageMicrophone" : "Baskuligi Mikrofon-stativon",
+        "Tutorial.Streaming.GreenScreenControl" : "Verd-kurtena Regilo",
```
Comment (Inferus25) [inline eo.json:1391]: "Regilo de verda ekrano". "Kurteno" doesn't work that way

File: eo.json [hunk 3]
```diff
@@ -1382,6 +1382,20 @@
         "Tutorial.Intro.HelpSite" : "Helpo",
 
         "Object.Mirror.Toggle" : "Loka Baskulo",
+        
+        "Tutorial.Streaming.StageControl" : "Sceneja Regilo",
```
Comment (Inferus25) [inline eo.json:1386]: "Regilo de scenejo" would be clearer. "Sceneja regilo" could mean a controller in the stage, and not a controller for the stage

**Review by Inferus25 — CHANGES_REQUESTED** [review]
Just a few changes according to the Mozilla localization guide we agreed upon. (only initial uppercase letter, and disctintion between -a vs ... de ...)
And line 1399 is missing. I would suggest "Baskuligi lum-ombrojn"

**Comment by Inferus25** [comment]
Once you make the -ante -> -anta change, I think you can request the merge.
We can take care of other tweaks later

**Comment by yellowcrash10** [comment]
> Just a few changes according to the Mozilla localization guide we agreed upon. (only initial uppercase letter, and disctintion between -a vs ... de ...)
> And line 1399 is missing. I would suggest "Baskuligi lum-ombrojn"

Perhaps I misunderstood (apologies if I did!), but I thought we only agreed on the -ante -> -anta change.

**Comment by Inferus25** [comment]
My bad. You are right. We only agreed on the -ante/-anta change. Please ignore my poor choosing of words.

However the guide has a fair point regarding the distinction between description and possession. "Dosiera grando" is not the same as "Grando de dosiero". I think we should take more care about that too. Or what do you think?

**Comment by Inferus25** [comment]
Hi, @yellowcrash10! How are you doing?
This PR has taken some time already to complete.
If it seems good to you, I think you can leave it with the main changes proposed at the beginning. The last ones are just details, rather than core material.

The overhaul is really awesome already :)

**Comment by yellowcrash10** [comment]
Sorry for the late response. I got busy suddenly last month and didn't have time to work on this. I'll just open this for @Frooxius to merge. As you said, we can work on the small details later.

**Comment by Frooxius** [comment]
Ah thank you for the overhaul, merging!

**Pull request merged**